libs/ledger-live-common/src/families/hedera/errors.ts

```diff
@@ -0,0 +1,3 @@
+import { createCustomErrorClass } from "@ledgerhq/errors";
+
+export const HederaAddAccountError = createCustomErrorClass("HederaAddAccountError");
```

Comment:
Prettier issue here as well it seems
https://github.com/LedgerHQ/ledger-live/actions/runs/3137430722/jobs/5095587144#step:10:2852